_content/papers/beyond-reasonable-doubt_vincent-eltschinger.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Beyond Reasonable Doubt? A Note on Dharmakīrti
+and Scepticism"
+authors:
+  - "Vincent Eltschinger"
+external_url: "https://philpapers.org/archive/ELTBRD.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1GrO0-Iiz75FIchradGLURiEpuCr7Ze2I/view?usp=sharing"
+course: epistemology
+tags:
+  - scepticism
+  - stoicism
+  - logic
+year: 2020
+booktitle: "Buddhism and Scepticism: Historical, Philosophical, and Comparative Perspectives"
```

Comment:
This book lives here: https://buddhistuniversity.net/content/monographs/buddhism-and-scepticism_hanner

So please use `from_book: buddhism-and-scepticism_hanner` instead 😁

(And, apparently, the top link for the book on Google:
![bookresults](https://github.com/buddhist-uni/buddhist-uni.github.io/assets/61610173/3d90528b-a955-44f5-918f-efae1ca364aa)
😲)